Source/Fuse.Controls.Native/SystemScreenConfigBase.uno

```diff
@@ -0,0 +1,129 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+using Fuse.Internal;
+
+namespace Fuse.Controls
+{
+	public class SystemScreenConfigBase : Behavior
+	{
+		public enum Visibility
+		{
+			None,
+			Minimal,
+			Full,
+		}
+
+		private static MiniList<SystemScreenConfigBase> rootedConfigs = new MiniList<SystemScreenConfigBase>();
```

Comment:
Actually, since you're only supposed to have one rooted Config you don't even need this list.